cocos/scene-graph/node.jsb.ts

```diff
@@ -1342,33 +1337,6 @@ nodeProto._ctor = function (name?: string) {
     this._euler = new Vec3();
 
     this._registeredNodeEventTypeMask = 0;
-
-    this.on(NodeEventType.CHILD_ADDED, (child) => {
-        this._children.push(child);
-    });
-
-    this.on(NodeEventType.CHILD_REMOVED, (child) => {
-        const removeAt = this._children.indexOf(child);
-        if (removeAt < 0) {
-            errorID(1633);
-            return;
-        }
-        this._children.splice(removeAt, 1);
-    });
-
-    this._onSiblingIndexChanged = function (index) {
-        const siblings = this._parent._children;
-        index = index !== -1 ? index : siblings.length - 1;
-        const oldIndex = siblings.indexOf(this);
-        if (index !== oldIndex) {
-            siblings.splice(oldIndex, 1);
-            if (index < siblings.length) {
-                siblings.splice(index, 0, this);
-            } else {
-                siblings.push(this);
-            }
-        }
-    }
```

Comment:
Move the `_onSiblingIndexChanged ` method to Node's prototype to avoid assigning a property each time  in `_ctor`.